docs/advanced/api_examples.yml

```diff
@@ -281,6 +281,29 @@
 
 -
   id:
+    "create container with environment"
+
+  doc: |
+    Create a new container.
+
+  request: |
+    POST /v1/configuration/containers HTTP/1.1
+
+    {"host": "%(NODE_0)s",
+     "name": "webserver",
+     "image": "nginx:latest",
+     "environment": {"CONFIG_FILE": "/etc/nginx/nginx.conf", "USER": "webserver"}}
```

Comment:
I was curious if these environment variables were actually supported by the official Nginx docker image. It looks like they're not.
 * https://github.com/nginxinc/docker-nginx/blob/master/Dockerfile

The `CONFIG_FILE` variable seems realistic. 
But the `USER` seems a little contrived. Why would an administrator care which user the webserver runs as inside the container. 
Maybe remove that one.